openshift/message-board/message-service/src/main/java/org/hibernate/demo/message/post/core/service/procedure/RemoteProcedureService.java

```diff
@@ -0,0 +1,27 @@
+package org.hibernate.demo.message.post.core.service.procedure;
+
+import javax.ejb.Stateless;
+import javax.inject.Inject;
+import javax.persistence.EntityManager;
+import javax.persistence.ParameterMode;
+import javax.persistence.StoredProcedureQuery;
+
+@Stateless
+public class RemoteProcedureService {
+
+	@Inject
+	private RemoteProcedureRegister register;
+
+	@Inject
+	private EntityManager em;
+
+	public Integer addToBoard(Long messageId) {
```

Comment:
What are you adding to the message board? Also whey this method starts with `add` but the stored procedure is an `Update*`?

Reply:
You're right, it isn't clear at all. I will change its name into `updateBoardWithMessage`. Maybe it is better. Isn't it?

Reply:
Sounds better